sickbeard/name_parser/parser.py

```diff
@@ -568,6 +619,10 @@ def is_anime(self):
             return True
         return False
 
+    @property
+    def is_proper(self):
+        return re.search(r'(^|[\. _-])(proper|repack)([\. _-]|$)', self.extra_info, re.I) is not None if self.extra_info else False
```

Comment:
This logic was just moved from one place to another. This is the current parser logic (not the guessit parser). I'm not changing/fixing the current parser